app/components/funding_widget_component.html.erb

```diff
@@ -52,20 +51,20 @@
             <div>
               <%= helpers.safe_format(funding_results) %>
 
-              <p>Tuition fee and maintenance loans are also available.</p>
+              <p>You can receive a scholarship or bursary alongside a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a>.</p>
```

Comment:
Doesn't it? I feel like because we're not specifically saying 'you can receive your **biology** scholarship or bursary', but instead saying 'you can receive **a** scholarship or bursary', it's applicable for every subject because it's not specific?

Reply:
@GDW7 Good to hear you opinion on this one!